### DISCUSSION TITLE
Welcome to webannoyances Discussions!

### DISCUSSION THREAD

**Discussion by yourduskquibbles** [yourduskquibbles/webannoyances#293] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Bearmancer** [comment]
Is this list being actively updated?

**Comment by NiteHackr** [comment]
I just clicked the link to add this to my uBlock Origin and it lists 63 errors in it.

**Comment by Mike-Bee1** [comment]
Excellent, no more annoying video pop-outs, thanks for the great work 👌👏
